alertmanager/multitenant.go

```diff
@@ -296,3 +399,32 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	}
 	userAM.router.ServeHTTP(w, req)
 }
+
+// GetStatusHandler returns the status handler for this multi-tenant
+// alertmanager.
+func (am *MultitenantAlertmanager) GetStatusHandler() StatusHandler {
+	return StatusHandler{
+		am: am,
+	}
+}
+
+// StatusHandler shows the status of the alertmanager.
+type StatusHandler struct {
+	am *MultitenantAlertmanager
+}
+
+// ServeHTTP serves the status of the alertmanager.
+func (s StatusHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	meshStatus := mesh.NewStatus(s.am.meshRouter.Router)
+	err := statusTemplate.Execute(w, meshStatus)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+}
+
+func printState(enabled bool) string {
```

Comment:
Is this kind of left over accidentally? The same function is defined in your function map above.